Runtime/Runner/SimvaManager.cs

```diff
@@ -135,7 +135,7 @@ public IAsyncOperation LoginAndSchedule()
                 .Catch(error =>
                 {
                     NotifyLoading(false);
-                    NotifyManagers(error.Message);
+                    NotifyManagers("Failed to Login with this token");
```

Comment:
Add a SimvaPlugin.Instance error log of the error for debugging.

Reply:
fixed